CONTRIBUTING.md

```diff
@@ -35,6 +35,8 @@ $ oc patch clusterversion/version --patch '{"spec":{"overrides":[{"group":"v1",
 ```
 
 2. Make your code changes and build the binary with `make build`.
+> While running `make build` if you come across any error something like
+` OS_GIT_MAJOR: unbound variable`, make sure that you have pulled all the tags from the [openshift/builder](https://github.com/openshift/builder) repo.
```

Comment:
I think that we should supply an example of the command to pull the tags

Reply:
Done